actions/README.md

```diff
@@ -0,0 +1,33 @@
+# slsa-verifier-installer GitHub Action
+
+This action installs the SLSA verifier so you can verify SLSA provenance in your workflows.
+
+For more information about `slsa-verifier`, refer to the [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see https://slsa.dev.
+
+## Usage
+
+This action requires using GitHub-provided Linux runners.
+
+Add the following snippet to your Github workflow YAML file:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer@main
+with:
+    verifier-release: v1.3.0 # optional
+    install-dir: /tmp/slsa-verifier # optional
+    use-sudo: false # optional
+```
+
+For a full example workflow, see `.github/workflows/test.yml`.
+
+### Optional Inputs
+
+| Input | Description |
+| --- | --- |
+| `verifier-release` | `slsa-verifier` version to use. Default value is `v1.3.0`. |
+| `install-dir` | Where to install `slsa-verifier` binary. Default value is `$HOME/slsa-verifier`. |
```

Comment:
nit: The action could just decide the location itself and add that location to the PATH. Example in setup-go:
https://github.com/actions/setup-go/blob/268d8c0ca0432bb2cf416faae41297df9d262d7f/src/main.ts#L34-L41

Also ideally the installer action would support installing multiple versions of the verifier in a matrix. I think Go does this by installing different versions of go in different locations, but each time it's installed the PATH is updated so the latest installed takes precedence. Ex:
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go#using-multiple-versions-of-go

Reply:
I'm also in favor of not exposing such an option, unless there is a compelling use case for it.

Reply:
Done